cloud/scope/machine.go

```diff
@@ -402,6 +402,16 @@ func (m *MachineScope) InstanceSpec(log logr.Logger) *compute.Instance {
 			Preemptible: m.GCPMachine.Spec.Preemptible,
 		},
 	}
+	if m.GCPMachine.Spec.ProvisioningModel != nil {
```

Comment:
Should we also explicitly set the `ProvisioningModel` to `Standard` when not null or is it implicitly set by GCP?

Reply:
I do believe GCP defaults to `STANDARD`. All documentation seems to be focused around specifying `SPOT` and otherwise ignoring the field. 

I also figured it was lower risk to maintain the existing behavior when the field is not set, allowing the GCP default to continue to be used.